alerting/mimir_alertmanager.go

```diff
@@ -0,0 +1 @@
+package alerting
```

Comment:
No on this stage - the design proposed that we first unify without modification so that we can import from both Grafana and Mimir as soon as possible.

Trying to unify before importing seems like an unnecessary risk, given, that we can unify as much as we want of both structures while still depending on two separate structures.